spec/reek/report/location_formatter_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../../spec_helper'
+require_relative '../../../lib/reek/report/formatter'
+require_relative '../../../lib/reek/smells/duplicate_method_call'
+require_relative '../../../lib/reek/smells/smell_warning'
+
+RSpec.describe 'location formatters' do
+  let(:warning) do
+    Reek::Smells::SmellWarning.new(Reek::Smells::DuplicateMethodCall.new,
+                                   lines:   [11, 9],
+                                   message: 'Foo#bar calls baz.qux 2 times',
+                                   source:  'string')
+  end
+
+  describe Reek::Report::BlankLocationFormatter do
+    describe '.format' do
+      it 'returns a blank String regardless of the warning' do
+        expect(described_class.format(warning)).to eq('')
+      end
+    end
+  end
+
+  describe Reek::Report::DefaultLocationFormatter do
+    describe '.format' do
+      it 'returns a prefix with sorted line numbers' do
+        expect(described_class.format(warning)).to eq('[9, 11]:')
```

Comment:
Can we broaden our test cases to more than two line numbers? With just 2 it's a 50:50 chance that is sorted correctly already :laughing: